nixos/modules/services/misc/pghero.nix

```diff
@@ -0,0 +1,138 @@
+{ config, pkgs, lib, utils, ... }:
+let
+  cfg = config.services.pghero;
+  settingsFormat = pkgs.formats.yaml { };
+  settingsFile = settingsFormat.generate "pghero.yaml" cfg.settings;
+in
+{
+  options.services.pghero = {
+    enable = lib.mkEnableOption (lib.mdDoc "PgHero service");
+    package = lib.mkPackageOptionMD pkgs "pghero" { };
+
+    listenAddress = lib.mkOption {
+      type = lib.types.str;
+      example = "[::1]:3000";
+      description = lib.mdDoc ''
+        `hostname:port` to listen for HTTP traffic.
+
+        This is bound using the systemd socket activation.
+      '';
+    };
+
+    extraArgs = lib.mkOption {
+      type = lib.types.listOf lib.types.str;
+      default = [ ];
+      description = lib.mdDoc ''
+        Additional command-line arguments for the systemd service.
```

Comment:
Can we link some upstream doc to find out more about the supported args?

Reply:
Done.